examples/README.md

```diff
@@ -0,0 +1,28 @@
+# HWP CLI Examples
+
+This directory contains examples demonstrating how to use the HWP CLI package.
```

Comment:
I'm pretty sure the examples folder will be holding ALL of our examples, not just CLI stuff.